### ISSUE TITLE
Epub validation error cuased by aria-describedby attribute

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/wax-prosemirror#4]
> **Migrated from GitLab**
> **Original Author:** Dione Mentis (`@DioneMentis`)
> **Original Link:** https://gitlab.coko.foundation/wax/wax-prosemirror/-/issues/123

## Expected behaviour

When "Long description" attribute isn't used in cofig, the attribute should be included

## Current behaviour

In the Ketty app we get an epub validation error: `The aria-describedby attribute must refer to elements in the same document (target ID missing)` this attribute isn't enabled for users to add. 

![Screenshot_2024-07-29_at_17.04.52](/uploads/73a1c0db2d6c89f4892286487b9dbf17/Screenshot_2024-07-29_at_17.04.52.png)

## Steps to reproduce

1. Add image in chapter
2. Try export to epub
3. Get error
4. Got back to image image and inspect 

```html
<img src="https://kotahi-s3.cloud68.co/test2-ketida/703ba07ac798_medium.jpeg?X-Amz-Algorithm=AWS4-HMAC-SHA256&amp;X-Amz-Credential=gbrqLPQZVrxwcsBaRztVkJTdLNqVOo%2F20240729%2Fus-east-1%2Fs3%2Faws4_request&amp;X-Amz-Date=20240729T145537Z&amp;X-Amz-Expires=86400&amp;X-Amz-Signature=90f15f9ff0bbf750f14aa4b0461f0ad3d2e63ef34d547b521589cd28d0193564&amp;X-Amz-SignedHeaders=host" alt="This is alt text" aria-describedby="1562dbee-7553-4df6-aa56-7975b5968141" aria-description="" data-id="abdde0c3-f315-448f-a9c4-107b9896d87b" data-fileid="0567f6bf-02cd-4ed2-8b8b-7631be460e1b" contenteditable="false" class="">
```

## Possible solution

The fix in wax should avoid adding aria-description if we haven't specified that we want to use image descriptions in the config

**Comment by yannisbarlas** [comment]
> **Migrated Comment**
> **Author:** Dione Mentis (`@DioneMentis`)
> **Date:** 2024-07-29T15:08:58.196Z

`@vukilelanga`  -- Grigor can give more info if necessary
